### ISSUE TITLE
Issue in dedup behaviour when dedup is enabled but the issue was resolved, closed or cancelled

### ISSUE THREAD

**Issue by nadiagutierrez89** [guilhemmarchand/TA-jira-service-desk-simple-addon#102]
Hello Guillem,

Your app is great! 
I was running a few tests about generating a ticket with dedup behaviour enabled, and detected an issue when you have two tickets with the same md5 hash and the first has a Status Category thats in "JIRA dedup excluded status categories".
In this case the key saved in KVstore for the second ticket is a random hash instead the MD5 Hash.
Looking for the problem in the code, i check "modalert_jira_service_desk_helper.py" and i found a missing tab in line 526.
I attached a print of the original line and the fixed line. I proved the functionality post fix and it work ok.

Apologies for my poor english.
Best regards!
![AddonJiraFixedl](https://user-images.githubusercontent.com/11739000/128244703-d3f57653-f873-4681-9803-ef7a01b76f3d.JPG)
![AddonJiraOriginal](https://user-images.githubusercontent.com/11739000/128244712-ab395740-5908-4cdf-963a-bff9a2f7fbf6.JPG)

**Comment by guilhemmarchand** [comment]
Hi @nadiagutierrez89 !

Thank you very much for reporting this, much appreciated ;-)
Ha ha no worries at all for your English, I am not a native speaker either!

All right, I see the issue indeed, exactly in this block:

```
            else:
                helper.log_info(
                    'jira_dedup: An issue with same md5 hash (' + str(jira_md5sum) + ') was found in the backlog '
                    'collection, as jira_dedup is not enabled a new issue '
                    'will be created, entry:={}'.format(response.text))
            jira_dedup_md5_found = True
```

This will be fixed for the Version 2.0.0 of the Add-on, which I am currently running with the latest validation steps.

Note that support for Splunk 7.2.x / Splunk 7.3.x is dropped and only Splunk 8.x will be supported.
Potentially I could drop a corrective release for this bug especially.

**Comment by nadiagutierrez89** [comment]
Thank you so much!!  

King regards :D
